kyaml/kio/filters/container.go

```diff
@@ -110,6 +113,10 @@ func (c *ContainerFilter) getArgs() []string {
 		args = append(args, "-v", fmt.Sprintf("%s:/local/:ro", c.mountPath))
```

Comment:
Consider building a list of mount points and including this in them.

Reply:
Actually maybe just get rid of this altogether

Reply:
Just so I don't misinterpret, get rid of what altogether exactly? I'm assuming you're talking about _both_ the mountPath field in the Container Filter struct and the line:

`args = append(args, "-v", fmt.Sprintf("%s:/local/:ro", c.mountPath))`

Reply:
Er, yes I meant that we should *keep* it, but have a single list of the mounted volumes, and append it to that list.